playbooks/roles/supervisor/files/pre_supervisor_checks.py

```diff
@@ -192,11 +194,13 @@ def edp_for_instance(instance_id):
         msg = "Failed to tag volumes associated with {}: {}".format(instance_id, str(e))
         print(msg)
 
+    ginkgo_xqueue = 0
+
     try:
         for service in services_for_instance(instance_id):
             if service in MIGRATION_COMMANDS:
                 # Do extra migration related stuff.
-                if service == 'xqueue' and args.xqueue_code_dir:
+                if service == 'xqueue' and args.xqueue_code_dir and ginkgo_xqueue:
```

Comment:
Since I hard-coded [`ginkgo_xqueue = 0` above](https://github.com/edx-olive/configuration/pull/18/files#diff-e8273f6396999e516fdd990a3d98e3b6R197), this will always evaluate to `False`.  I chose to do it this way to reduce the diff on this file, since it's used for all the other service deployments too, and I didn't want to inadvertently break them.  Will add a comment to this effect to make this clearer.

Reply:
@pomegranited Ah OK, I understand the current approach better now: The steps in this first branch of the conditional are only needed for the Ginkgo version of XQueue. We're using the Hawthorn version of XQueue for Campus, and so the alternative to the current approach would have been to remove the steps that are only relevant for Ginkgo completely. This would have increased the diff as well as the chances of breaking something for both XQueue and the other services using this script.